<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VoiceAlarmDialog</class>
 <widget class="QDialog" name="VoiceAlarmDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>409</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="SourceCB">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>52</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Source</string>
   </property>
  </widget>
  <widget class="QComboBox" name="FunctionCB">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>-------</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>v&gt;val</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>v&lt;val</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>|v|&gt;val</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>|v|&lt;val</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>v~=val</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>v=val</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>52</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Function</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="ValueSB">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>77</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-32000</number>
   </property>
   <property name="maximum">
    <number>32000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>52</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Value</string>
   </property>
  </widget>
  <widget class="QComboBox" name="SwitchCB">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>52</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Switch</string>
   </property>
  </widget>
  <widget class="QComboBox" name="RateCB">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ON</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OFF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BOTH</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>21</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>23</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>24</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>25</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>26</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>27</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>28</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>29</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>52</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>52</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Haptic</string>
   </property>
  </widget>
  <widget class="QComboBox" name="HapticCB">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>-------</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Haptic1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Haptic2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Haptic3</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="FileTypeCB">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>340</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>------</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Number</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Audio</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>52</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>File Type</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="FileNumberSB">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>380</y>
     <width>77</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-32000</number>
   </property>
   <property name="maximum">
    <number>32000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>52</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FileName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>380</y>
     <width>97</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>8</number>
   </property>
  </widget>
  <widget class="QComboBox" name="AudioCB">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>380</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>------</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Name</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Number</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Audio</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="TelemValue">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Play Source</string>
   </property>
  </widget>
  <widget class="QComboBox" name="PlaySourceCB">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>No</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Before</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>After</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="MuteCB">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>300</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Play</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Mute</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>On No Telemetry</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>VoiceAlarmDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>VoiceAlarmDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
